examples/preprocessing/eog_regression.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+"""
+=======================================
+Reduce EOG artifacts through regression
+=======================================
+
+Reduce artifacts by regressing the EOG channels onto the rest of the channels
+and then subtracting the EOG signal.
+
+This is a quick example to show the most basic application of the technique.
+See the `tutorial<_tut-artifact-regression>`_ for a more thorough explanation
+that demonstrates more advanced approaches.
+
+Authors: Marijn van Vliet <w.m.vanvliet@gmail.com>
+
+License: BSD (3-clause)
+"""
+# %%
+# Do imports and load the MNE-Sample data.
+import warnings
+
+import mne
+from mne.datasets import sample
+from mne.preprocessing import EOGRegression
+from matplotlib import pyplot as plt
+
+print(__doc__)
+
+data_path = sample.data_path()
+raw_fname = data_path / 'MEG' / 'sample' / 'sample_audvis_filt-0-40_raw.fif'
+
+# Read raw data
+raw = mne.io.Raw(raw_fname, preload=True)
```

Comment:
TIL you can pass a fname to the Raw constructor.  I'm always using `mne.io.read_raw_fif` for this.  Your way is simpler, maybe all examples/tutorials should do it this way.

Reply:
I hadn't even noticed. I actually disagree, because the `read_raw_*` interface is more consistent. This only works for `.fiff` files and nothing else.

Reply:
Ah ok